core/src/banking_stage.rs

```diff
@@ -567,6 +567,7 @@ impl BankingStage {
         };
 
         const INTERVAL_MS: u64 = 100;
+        // 12 MB outbound limit per second
         const MAX_BYTES_PER_SECOND: usize = 10_000 * 1200;
```

Comment:
This const is a little funky
`const MAX_BYTES_PER_SECOND: usize = 12_000_000;`
would be easier to read